Command/CreateUserCommand.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\Command;
+
+use SumoCoders\FrameworkMultiUserBundle\User\UserRepository;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class CreateUserCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('sumocoders:multiuser:create')
+            ->setDescription('Create a user entity')
+            ->addArgument(
+                'username',
+                InputArgument::REQUIRED,
+                'The username of the user'
+            )
+            ->addArgument(
+                'password',
+                InputArgument::REQUIRED,
+                'The password for the user'
+            )
+            ->addArgument(
+                'displayName',
+                InputArgument::REQUIRED,
+                'The display name for the user'
+            )
+            ->addOption(
+                'class',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'The class off the user',
+                'SumoCoders\FrameworkMultiUserBundle\User\User'
```

Comment:
Het is mss toffer om hier de mogelijke opties al te geen en mensen te laten kiezen daartussen met tab autocompletion?